<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Config</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkBoxConfigSmoothPlot">
              <property name="text">
               <string>Smooth Plot</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxConfigAntialias">
              <property name="text">
               <string>Antialias filtering</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Connect">
         <attribute name="title">
          <string>Connect</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBoxWirelessInterface">
            <property name="title">
             <string>Wireless interface:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButtonBTdiscoverDevices">
                 <property name="text">
                  <string>Discover BT</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QRadioButton" name="radioButtonWiFi">
                 <property name="text">
                  <string>WiFi</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="labelBTstatus">
                 <property name="text">
                  <string>Status:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QRadioButton" name="radioButtonBT">
                 <property name="text">
                  <string>Bluetooth</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QComboBox" name="comboBoxBTdevices"/>
               </item>
               <item row="5" column="0">
                <widget class="QPushButton" name="pushButtonBTconnect">
                 <property name="text">
                  <string>Connect BT</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>IP Address:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditWiFiAddress">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="5" column="2">
                <widget class="QPushButton" name="pushButtonWiFiConnect">
                 <property name="text">
                  <string>Connect WiFi</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLabel" name="labelWiFiStatus">
                 <property name="text">
                  <string>Status:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="labelWiFiPort">
                   <property name="text">
                    <string>Port:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditWiFiPort">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="6" column="2">
                <widget class="QPushButton" name="pushButtonWiFiDisconnect">
                 <property name="text">
                  <string>Disconnect WiFi</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QPushButton" name="pushButtonBTdisconnect">
                 <property name="text">
                  <string>Disconnect BT</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>AT</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="lineEditAT"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonATSend">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPlainTextEdit" name="plainTextEditAT"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Plot</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>radioButtonBT</tabstop>
  <tabstop>radioButtonWiFi</tabstop>
  <tabstop>lineEditWiFiAddress</tabstop>
  <tabstop>lineEditWiFiPort</tabstop>
  <tabstop>pushButtonWiFiConnect</tabstop>
  <tabstop>pushButtonWiFiDisconnect</tabstop>
  <tabstop>pushButtonBTdiscoverDevices</tabstop>
  <tabstop>comboBoxBTdevices</tabstop>
  <tabstop>pushButtonBTconnect</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEditAT</tabstop>
  <tabstop>pushButtonATSend</tabstop>
  <tabstop>plainTextEditAT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
